Content.Client/Parallax/ParallaxControl.cs

```diff
@@ -17,27 +19,48 @@ public sealed class ParallaxControl : Control
     [Dependency] private readonly IParallaxManager _parallaxManager = default!;
     [Dependency] private readonly IRobustRandom _random = default!;
 
-    [ViewVariables(VVAccess.ReadWrite)] public Vector2 Offset { get; set; }
+    private string _parallaxPrototype = "FastSpace";
+
+    [ViewVariables(VVAccess.ReadWrite)] public float SpeedX { get; set; } = 0.0f;
+    [ViewVariables(VVAccess.ReadWrite)] public float SpeedY { get; set; } = 0.0f;
+    [ViewVariables(VVAccess.ReadWrite)] public float ScaleX { get; set; } = 1.0f;
+    [ViewVariables(VVAccess.ReadWrite)] public float ScaleY { get; set; } = 1.0f;
+    [ViewVariables(VVAccess.ReadWrite)] public string ParallaxPrototype
+    {
+        get => _parallaxPrototype;
+        set
+        {
+            _parallaxManager.UnloadParallax(_parallaxPrototype);
```

Comment:
Isn't this gonna break an existing loaded usage of this parallax? There's no ref counting here.